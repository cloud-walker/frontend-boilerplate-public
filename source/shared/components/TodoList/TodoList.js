import React, {PropTypes} from 'react'

const Component = ({todos}) =>
  <ul>
    {todos.map((todo, i) =>
      <li key={i}>{todo}</li>
    )}
  </ul>

Component.displayName = 'TodoList'
Component.propTypes = {
  todos: PropTypes.array.isRequired,
}

export default Component
