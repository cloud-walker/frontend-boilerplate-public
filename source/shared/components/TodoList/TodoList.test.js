import React from 'react'
import {shallow} from 'enzyme'
import TodoList from './index'

test('<TodoList/>', () => {
  expect(shallow(<TodoList todos={[]}/>)).toMatchSnapshot()
})
