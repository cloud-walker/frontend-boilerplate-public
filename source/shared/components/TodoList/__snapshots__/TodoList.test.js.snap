exports[`test <TodoList/> empty 1`] = `null`;

exports[`test <TodoList/> with some todos 1`] = `
<ul>
  <li>
    Clean the Kitchen
  </li>
  <li>
    Go out for a walk
  </li>
</ul>
`;
